scootapi/client/run_job_cmd.go

```diff
@@ -8,11 +8,13 @@ import (
 )
 
 func makeRunJobCmd(c *Client) *cobra.Command {
-	return &cobra.Command{
+	r := &cobra.Command{
 		Use:   "run_job",
 		Short: "run a job",
 		RunE:  c.runJob,
 	}
+	r.Flags().StringVar(&c.dialer.addr, "addr", "localhost:9090", "address to connect to")
```

Comment:
Can you add a setter in Dialer to signal our intent to reset the addr externally?